src/openrct2-ui/interface/InGameConsole.cpp

```diff
@@ -237,15 +237,13 @@ void InGameConsole::WriteLine(const std::string& input, FormatToken colourFormat
 
 void InGameConsole::Invalidate() const
 {
-    gfx_set_dirty_blocks({ { _consoleLeft, _consoleTop }, { _consoleRight, _consoleBottom } });
+    gfx_set_dirty_blocks({ _consoleTopLeft, _consoleBotRight });
 }
 
 void InGameConsole::Update()
 {
-    _consoleLeft = 0;
-    _consoleTop = 0;
-    _consoleRight = context_get_width();
-    _consoleBottom = 322;
+    _consoleTopLeft = { 0, 0 };
+    _consoleBotRight = { context_get_width(), 322 };
```

Comment:
This is just silly :-P

```suggestion
    _consoleBottomRight = { context_get_width(), 322 };
```